docs/en/connector-v2/sink/PostgreSql.md

```diff
@@ -125,6 +125,7 @@ transform {
 
 sink {
     jdbc {
+       # if you would have json or jsonb type insert please add jdbc url stringtype=unspecified option
```

Comment:
If the user uses JSON-related types, tell it how to configure the url information, please prompt in the url Description in Options
![image](https://github.com/apache/seatunnel/assets/76689593/282a597d-c299-49d7-8547-a1ca214c8d4a)

Reply:
done @zhilinli123 PTAL